actionview/lib/action_view/helpers/asset_tag_helper.rb

```diff
@@ -35,18 +35,37 @@ module AssetTagHelper
       # When the Asset Pipeline is enabled, you can pass the name of your manifest as
       # source, and include other JavaScript or CoffeeScript files inside the manifest.
       #
+      # ==== Options
+      #
+      # When the last parameter is a hash you can add HTML attributes using that
+      # parameter. That options supports the following keys as well:
+      #
+      # * <tt>:extname</tt>  - Append a extention to the generated url unless the extension
+      #   already exists. This only applies for relative urls.
+      # * <tt>:protocol</tt>  - Sets the protocol of the generated url, this option only
+      #   apply when a relative url and +host+ options are provided.
```

Comment:
There's a grammatical error in this sentence. It should be "this option only applies when".